manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.11.0

```

```suggestion
PackageVersion: 6.11.0.47988
```
Should match control panel